R-packages/covidcast/vignettes/correlation-utils.Rmd.orig

```diff
@@ -0,0 +1,195 @@
+---
```

Comment:
Is this file meant to be included in this PR?

Reply:
Yes, see comment #470. This is the "raw" code that creates a pre-build correlation-utils.Rmd to speed up the installation time. It needs an odd extension so that R does not try to build it on installation.